node/src/main/scala/org/bitcoins/node/SpvNodeCallbacks.scala

```diff
@@ -0,0 +1,15 @@
+package org.bitcoins.node
+
+import org.bitcoins.node.networking.peer.DataMessageHandler._
+
+/**
+  * Callbacks for responding to events in the SPV node.
+  * The approriate callback is executed whenver the node receives
+  * a `getdata` message matching it.
+  *
+  */
+case class SpvNodeCallbacks(
```

Comment:
So I really like this idea! This really provides loose coupling of our various modules.